samples/net/wifi/boards/esp32.conf

```diff
@@ -0,0 +1,14 @@
+CONFIG_WIFI=y
+CONFIG_HEAP_MEM_POOL_SIZE=98304
+
+CONFIG_NETWORKING=y
+CONFIG_NET_L2_ETHERNET=y
```

Comment:
Shouldn't this be CONFIG_NET_L2_WIFI_MGMT?

Reply:
No, it still uses L2_ETHERNET layer.